src/style/variables.less

```diff
@@ -1,4 +1,4 @@
-@header-height: 3.5rem; // 56px
+@header-height: 5rem; // 80px
```

Comment:
Definitely don't want to do this, it increases the header size significantly.

Instead, increase the padding on the main content:

https://github.com/preactjs/preact-www/blob/b04ccc87e480c950a8a55b2a376d24dea09dda36/src/style/index.less#L93

```diff
-padding-top: @header-height;
+padding-top: calc(@header-height + 2.5rem);
```

Reply:
updated!